kotlin-runtime/ftl-generator/src/main/kotlin/xyz/block/ftl/generator/ModuleGenerator.kt

```diff
@@ -162,7 +153,7 @@ class ModuleGenerator() {
       }
 
       type.verbRef != null -> ClassName("xyz.block.ftl.v1.schema", "VerbRef")
```

Comment:
do we need to make this change for VerbRef too?

Reply:
I'm actually not sure what we should do with `VerbRef`. Do we generate `VerbRef` things?